src/assets/js/utils/listeners.js

```
@@ -0,0 +1,37 @@
+		target.removeEventListener(event, handler);
+	});
+};
+
+module.exports.addManagedListener = addManagedListener;
+
+module.exports.screenWidgetListener = (component) => {
+	addManagedListener(component, window, 'resize', debounce(throttle(() => component.set('screenWidth', window.innerWidth), 200)));
+};
+
+module.exports.stickySidebarScrollListener = (component) => {
+	let sidebar = document.querySelector('.page-content_side-menu');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'sidebar' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)